gatsby-browser.js

```diff
@@ -116,6 +117,9 @@ export const onInitialClientRender = () => {
             if (!isProduction()) LocalStore.set('i18n', 'ach')
         }
 
+        if (window.location.pathname.includes(besquare_signup_url))
```

Comment:
You can add `isbrowser() && window.location.pathname.includes(besquare_signup_url)`

Reply:
The if statement is already inside the **if (is_browser)** condition from above so I don't think that's necessary

Reply:
Okay got it.